lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -519,3 +569,116 @@ def test_skip_entrance_exame_gating(self):
         )
         for toc_section in self.expected_unlocked_toc:
             self.assertIn(toc_section, unlocked_toc)
+
+    def test_entrance_exam_gating_for_staff(self):
```

Comment:
I agree with @ziafazal here -- there are many tests in the suite which cover the 'no entrance exam configured' case, so I think we can focus on covering the active workflows here -- of course we need to ensure we have 100% coverage with the tests.